pkg/image/trigger/deploymentconfigs/deploymentconfigs.go

```diff
@@ -187,9 +193,6 @@ func UpdateDeploymentConfigImages(dc *appsapi.DeploymentConfig, tagRetriever tri
 			glog.V(4).Infof("trigger %#v in deployment %s is not resolveable", p, dc.Name)
 			return nil, false, nil
 		}
-		if ref == p.LastTriggeredImage {
```

Comment:
why is this going away?

Reply:
because it will stop DC...image from being reconciled. If someone overwrites it manually, this will prevent reconciliation which should have been done